src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -108,6 +109,15 @@ const TimeEntryForm = (props) => {
       .catch((err) => {});
   }, []);
 
+  useEffect(() => {
+    axios
+      .get(`${ApiEndpoint}/tasks/userProfile?members=${userId}`)
```

Comment:
You should define an endpoint for this URL (I think in URL.js) so that it is reusable. If one already exists, you should use that.

Reply:
**Addressed.**